UI/EncyclopediaDetailPanel.cpp

```diff
@@ -2287,8 +2287,29 @@ namespace {
         for (SpeciesManager::iterator it = species_manager.begin();
                 it != species_manager.end(); ++it)
         {
-            if (it->second && (it->second->Tags().find("CTRL_ALWAYS_REPORT") != it->second->Tags().end()))
-                    species_names.insert(it->first);
+            if (it->second) {
```

Comment:
prefer

    if (!it->second)
        continue;

and then don't indent the following code.